static/app/types/profiling.d.ts

```diff
@@ -1,4 +1,75 @@
 declare namespace Profiling {
+  type SentrySampledProfileSample = {
+    stack_id: number;
+    thread_id: string;
+    elapsed_since_start_ns: string;
+    queue_address?: string;
+  };
+
+  type SentrySampledProfileStack = number[];
+
+  type SentrySampledProfileFrame = {
+    name?: string;
+    instruction_addr?: string;
+    line?: number;
+    file?: string;
```

Comment:
@phacops we support `column` too right?

Reply:
Now we do, yes. Technically `colno`, aliased to `column`.